docs/userguide/containers/dockerizing.md

```diff
@@ -24,147 +24,131 @@ Running an application inside a container takes a single command: `docker run`.
 
 ## Run a Hello world
 
-Let's try it now.
+Let's run a hello world container.
```

Comment:
@moxiegirl yes, most of this information is in Getting Started (https://docs.docker.com/mac/step_one) and seems redundant.  The section and the Run a simple application section could be removed.

Iv'e seen "let's" a few times and thought it might be part of the "style" - I totally agree with removing it and only using second person.